### ISSUE TITLE
bend90.py fails to run

### ISSUE THREAD

**Issue by jonbenfri** [stanfordnqp/spins-b#66]
Summary:
Running bend90.py results in a missing module error due to a missing init file. Adding the init file solves the issue. See #65.

Steps to reproduce:
1. Git clone most recent master branch 9cd1510.
2. Install using: pip install ./spins-b
3. Change directory to examples/goos/bend90.
4. Try to run bend90: python bend90.py
5. Error message: ModuleNotFoundError: No module named 'spins.goos_sim'

**Comment by snapo** [comment]
same issue...

**Comment by teytaud** [comment]
@snapo @jonbenfri  Maybe try (if linux) "ln -s /path/to/spins-b/spins ." before running "python bend90.py".

**Comment by jonbenfri** [comment]
The solution that worked for me was to create empty files named `__init__.py` and place them in the appropriate directories. In Linux you can create the empty files as follows):
```bash
cd /path/to/spins-b
touch spins/__init__.py
touch spins/goos_sim/__init__.py
```

I submitted a pull request for the solution here: #65

**Comment by jonbenfri** [comment]
Fixed by #73. Tested using system-wide Python 3.9.7 on Ubuntu 21.10.
